src/components/MyActivityCard.tsx

```diff
@@ -0,0 +1,78 @@
+import "./MyActivityCard.scss"
+
+import React, { ReactElement } from "react"
+import { HistoricalPoolDataType } from "../hooks/useHistoricalPoolData"
+import { formatUnits } from "@ethersproject/units"
+import { useTranslation } from "react-i18next"
+
+interface Props {
+  historicalPoolData: HistoricalPoolDataType | null
+}
+
+function MyActivityCard({ historicalPoolData }: Props): ReactElement | null {
+  const { t } = useTranslation()
+
+  if (!historicalPoolData) return null
+
+  const historicalFormattedData = historicalPoolData
+    ? {
+        totalDepositsBTC: parseFloat(
+          formatUnits(historicalPoolData.totalDepositsBTC, 36),
+        ).toFixed(5),
+        totalWithdrawalsBTC: parseFloat(
+          formatUnits(historicalPoolData.totalWithdrawalsBTC, 36),
+        ).toFixed(5),
+        totalProfitBTC: parseFloat(
+          formatUnits(historicalPoolData.totalProfitBTC, 36),
+        ).toFixed(5),
+        totalDepositsUSD: parseFloat(
+          formatUnits(historicalPoolData.totalDepositsUSD, 36),
+        ).toFixed(),
+        totalWithdrawalsUSD: parseFloat(
```

Comment:
Moving toFixed to HTML entirely for all the different values would be better for consistency here I think.